packages/client/test/integration/test-node/test/main.spec.ts

```diff
@@ -217,6 +240,13 @@ describe('Events API', () => {
 
 describe('Setting configuration', () => {
```

Comment:
This test needs to be deleted. We are testing this in Torii api tests.

Reply:
Deleted